### DISCUSSION TITLE
Delaying reproduction in initialisation

### DISCUSSION THREAD

**Discussion by ellalewi547** [RangeShifter/RangeshiftR-tutorials#46] [Q&A]
Hi again, 

I am initialising stage 3 (adults) as a reintroduction scenario and I wondered if there was any way of delaying the reproduction for 2 years to reflect them settling into a new environment? I would initialise them at a non-breeding age (stage 2) but stage 2 individuals in my model disperse and I want to model them not dispersing from their release site for a year.

Thanks in advance

**Comment by JetteReeg** [comment]
Hi Ella,

currently there are no options to delay reproduction nor dispersal in the first years after initialisation. 

But I am wondering, why it would be important (simulation wise) that your individuals show no reproduction in the first years? I guess it should be ok to initialise stage 3 individuals. 

(Please be aware that we cannot provide much help conceptualizing RangeShiftR models and mainly help with technical issues.)

**Comment by ellalewi547** [comment]
Okay thank you for the help :)
